package/gluon-core/luasrc/lib/gluon/upgrade/140-firewall-rules

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/lua
+-- TODO: replace REJECT by DROP. only use REJECT for debugging
```

Comment:
I disagree, REJECT is preferable; a proper unreachable message is much nicer than connection attempts just hanging.. The *log* options should be removed though, so we can get this ready for merging.

Reply:
done.